### DISCUSSION TITLE
Upgrading package for astronomer-providers[dbt.cloud] from 1.9.0 to 1.16.0

### DISCUSSION THREAD

**Discussion by miodeqqq** [astronomer/astronomer-providers#1219] [Q&A]
After upgrading **astronomer-providers[dbt.cloud]** package from version **1.9.0** to **1.16.0** all of my DBT jobs started to fail. Did I miss something in my config ? 

```python3
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/astronomer/providers/dbt/cloud/operators/dbt.py", line 96, in execute_complete
    raise AirflowException(event["message"])
airflow.exceptions.AirflowException: Cannot connect to host https:443 ssl:default [Name or service not known]
```

**Comment by miodeqqq** [comment]
cc: @pankajkoti

**Reply by pankajkoti** [reply]
hi, could you please share some more logs from triggerer or task? Also, could you share your DAG if possible?

Based on the available error message, it appears it is not able to connect to your DBT cloud, no?

The example DAG https://github.com/astronomer/astronomer-providers/blob/main/astronomer/providers/dbt/cloud/example_dags/example_dbt_cloud.py works in our integrations tests that we run on a daily basis, worked today as well.

**Reply by pankajkoti** [reply]
There have been no breaking changes as far as I checked. Has something else changed too in your configuration? The connection/hosts/creds perhaps?

**Reply by miodeqqq** [reply]
Not really, this is our prod environment and the only change was here to upgrade the package 🤔

**Reply by pankajkoti** [reply]
Okay. It's difficult to debug from the limited info. Will need the DAG to repoduce the issue. Would it be possibe to share more logs from Triggerer? Which version of Airflow are you on? The triggerer would have more logs when you run the DAG

**Reply by pankajkoti** [reply]
If you'd like to share more details in a closed fashion my work email ID is pankaj.koti@astronomer.io

**Reply by pankajkoti** [reply]
hi @miodeqqq we're taking forward the discussion in the support ticket https://astronomer.zendesk.com/agent/tickets/23499

**Reply by miodeqqq** [reply]
Hey @pankajkoti, I apologize for the delay in response. It was necessary for me to consult with the entire Team before proceeding and providing further information. I appreciate your patience and understanding in this matter. And yeah - we've raised the support ticket, thanks!

**Reply by pankajkoti** [reply]
Not a problem @miodeqqq . I will close this discussion here as we have a ticket for it.
